### PR TITLE
Update the documentation of Integrating Scrapy doc page

### PR DESCRIPTION
**Pull request by vdusek**

### CODE REVIEW

File: docs/integrating-scrapy.md [hunk 1]
```diff
@@ -140,6 +140,58 @@ The [scheduler](https://docs.scrapy.org/en/latest/topics/scheduler.html) is a co
...
+[AsyncIO](https://docs.python.org/3/library/asyncio.html), which brings some complications on the table.
+
+Due to the asynchronous nature of the Actors, all of their code is executed as a coroutine inside the `asyncio.run`.
+In order to execute Scrapy code inside an Actor, following the section
+[Run Scrapy from a script](https://docs.scrapy.org/en/latest/topics/practices.html?highlight=CrawlerProcess#run-scrapy-from-a-script)
+from the official Scrapy documentation, we need to invoke a
+[CrawlProcess.start](https://github.com/scrapy/scrapy/blob/2.11.0/scrapy/crawler.py#L393:L427)
+method. This method triggers Twisted's event loop, also known as a reactor.
+Consequently, Twisted's event loop is executed within AsyncIO's event loop.
+On top of that, when employing AsyncIO code in Spiders or other components, it necessitates the creation of a new
```
Comment (TC-MO) [inline docs/integrating-scrapy.md:159]: Is there any particular reason why spiders is capitalized here?

File: docs/integrating-scrapy.md [hunk 2]
```diff
@@ -140,6 +140,58 @@ The [scheduler](https://docs.scrapy.org/en/latest/topics/scheduler.html) is a co
...
+### Asynchronous code in spiders and other components
+
+Scrapy asynchronous execution is based on the [Twisted](https://twisted.org/) library, not the
+[AsyncIO](https://docs.python.org/3/library/asyncio.html), which brings some complications on the table.
+
+Due to the asynchronous nature of the Actors, all of their code is executed as a coroutine inside the `asyncio.run`.
+In order to execute Scrapy code inside an Actor, following the section
+[Run Scrapy from a script](https://docs.scrapy.org/en/latest/topics/practices.html?highlight=CrawlerProcess#run-scrapy-from-a-script)
+from the official Scrapy documentation, we need to invoke a
+[CrawlProcess.start](https://github.com/scrapy/scrapy/blob/2.11.0/scrapy/crawler.py#L393:L427)
```
Comment (TC-MO) [inline docs/integrating-scrapy.md:156]:
```suggestion
[`CrawlProcess.start`](https://github.com/scrapy/scrapy/blob/2.11.0/scrapy/crawler.py#L393:L427)
```

Comment (TC-MO) [inline docs/integrating-scrapy.md:156]:
```suggestion
[`CrawlProcess.start`](https://github.com/scrapy/scrapy/blob/2.11.0/scrapy/crawler.py#L393:L427)
```

**Review by TC-MO — CHANGES_REQUESTED** [review]

File: docs/integrating-scrapy.md (hunk 1 above)
Comment (vdusek) [reply docs/integrating-scrapy.md]: it's a mistake, thanks

**Review by TC-MO — APPROVED** [review]

**Pull request merged**